components/style/lib.rs

```diff
@@ -72,6 +74,11 @@ extern crate time;
 extern crate url;
 extern crate util;
 
+#[cfg(feature = "gecko")]
+#[path = "./gecko_string_cache/mod.rs"]
+#[allow(unsafe_code)]
+#[macro_use] pub mod string_cache;
```

Comment:
`mod string_cache` only _exists_ for Gecko. There's a cfg four lines above this. Servo uses a crate.